Documentation/LanguageServer/colorization.md

```diff
@@ -0,0 +1,87 @@
+
+# VS Code C/C++ Extension - Enhanced Colorization
+
+The VS Code C/C++ extension now supports lexical/syntactic and semantic colorization, using IntelliSense.
+
+# Theming
+
+Colors can be associated using the existing support for theming and color customization in VS Code.  Documentation on Theming in VS Code can be found <a HREF="https://code.visualstudio.com/docs/getstarted/themes">here</a>.
```

Comment:
Is there a reason you are using an HTML anchor instead of a markdown link?

Reply:
Nope, just inexperience with markdown.  :)